src/screens/Settings/components/GeneralSettings/index.tsx

```diff
@@ -252,6 +256,12 @@ export default function GeneralSettings({
           <ToggleSwitch value={exitToTray} handleChange={toggleTray} />
         </span>
       </span>
+      <span className="setting">
+        <span className="toggleWrapper">
+          {t('setting.start-in-tray')}
```

Comment:
To avoid rendering a empty label, its good to have a fallback, so you can change this line to this one:
```suggestion
          {t('setting.start-in-tray', 'Start Minimized)}
```

Reply:
Fixed!